hprofdata/hprofdata.proto

```diff
@@ -115,6 +115,7 @@ message HProfClassDump {
 
   // Constant pool entry (appears to be unused according to heapDumper.cpp).
   message ConstantPoolEntry {
+    uint32 constant_pool_index = 3;
```

Comment:
I used the new field index as 3 for backward compatibility. But if you don't need to care about the backward compatibility at this time, I can re-numbering the index.

Reply:
Nope. It's ok. We have no reason to break the compatibility.